### PR TITLE
polish info button

### PR DESCRIPTION
**Pull request by emely3h**
ticket: https://www.notion.so/polish-info-button-to-make-it-a-bit-more-professional-bette-r-link-repository-readme-858e92a56d1c42cca1274de46d23c90e

### CODE REVIEW

File: src/components/organisms/OEWHeader/OEWHeader.tsx [hunk 1]
```diff
@@ -90,12 +90,28 @@ const OEWHeader: React.FC<OEWHeaderProps> = ({ logo, isOpen, regionProps, handle
...
                         <p className="text-start">
-                            The <a href="https://www.oceanecowatch.org/">Ocean Eco Watch</a> is a map highliting potential locations of floating
-                            marine debris. We use data from the sentinel-2 satellite. Click on a location to get the detailed analysis of the area.
-                            Each point on the map corresponds to the probability of present marine debris and represents an area spanning 10 m².
+                            <a href="https://www.oceanecowatch.org/">Ocean Eco Watch</a> is an interactive map highlighting potential locations of
+                            floating marine debris in various coastal areas. Utilizing data from the ESA's Sentinel-2 satellite, our map identifies
+                            and analyzes debris hotspots.
+                            <br></br>
+                            To explore, click on any red dot and zoom in to see a detailed marine debris analysis of the area. Each point represents a
+                            10 m² area with an estimated probability of marine debris presence.
```
Comment (tomADC443) [inline src/components/organisms/OEWHeader/OEWHeader.tsx]:
```suggestion
                            10m x 10m area with an estimated probability of marine debris presence.
```

Comment (tomADC443) [reply src/components/organisms/OEWHeader/OEWHeader.tsx]: or 100m^2

**Comment by tomADC443** [comment]
awesome!  Nois! way better then before! 

just one thing I noticed in the old text. (see commit) 

> represents a 10 m² area with an estimated probability of marine debris presence.

either 10m x10m or 100m² :D

**Review by tomADC443 — APPROVED** [review]

**Comment by emely3h** [comment]
thx, same mistake as always:D

**Pull request merged**